packages/terra-hyperlink/tests/wdio/hyperlink-spec.js

```diff
@@ -0,0 +1,120 @@
+/* global browser, Terra, before */
+const viewports = Terra.viewports('tiny', 'medium', 'large');
```

Comment:
There isn't really any responsive behavior to this, should I trim this down to run in just 1 viewport?

Reply:
Talked in meeting today, plan to drop this down to just use tiny viewport